### DISCUSSION TITLE
Exclude required words not working - Solved

### DISCUSSION THREAD

**Discussion by Maydeedvd** [pymedusa/Medusa#9390] [Q&A]
Hi,
First thanks you for this amazing tool ^^

I have an issue : i'm use by default (required words) th search with the words "VOSTFR" and  "MULTI" because i'm french.
There are 3 series i watch in VO only, so i slide the Exclude required words button in advanced and it's not working.
In LOG, it's say :  Ignoring "myserieinVO" based on required words filter: ['VOSTFR', 'MULTI'].
Thanks

**To Reproduce**
1. Go to EDIT
2. Click on ADVANCED
3. Slide the button Exclude required words
4. No change

**Comment by medariox** [comment]
The title needs to match the words. ['VOSTFR', 'MULTI'] doesn't match 'myserieinVO'.

**Reply by Maydeedvd** [reply]
Hi, the VOSTFR'/'MULTI' filters works. It's not the issue. 
The issues is the button  "Exclude required words" in serie's paramaters : it's not worling

**Reply by medariox** [reply]
You need to post debug logs as stated when opening an issues, otherwise we can't help you.

**Comment by Maydeedvd** [comment]
This is the log, you can see the show still use the global preference (VOSTFR/MULTI) and not use the series preferences (exclude required words)....
Ignoring Grown-ish.S03E17.720p.WEB.H264-CAKES[rartv] based on required words filter: ['VOSTFR', 'MULTI']
[log.txt](https://github.com/pymedusa/Medusa/files/6222797/log.txt)

**Reply by p0psicles** [reply]
Can you share some screenshots of your config?

**Comment by Maydeedvd** [comment]
Yes, of course, thanks for your time 
![MEDUSA 1](https://user-images.githubusercontent.com/59835411/112864521-2be8ee80-90b8-11eb-9b72-82cd9ae84e3b.JPG)
![MEDUSA 2](https://user-images.githubusercontent.com/59835411/112864526-2c818500-90b8-11eb-9552-3cab5b0fae95.JPG)

**Reply by p0psicles** [reply]
You need to add those required words there to exclude them. You only flipped the toggle. But it doesn't know which words you want to exclude

**Reply by Maydeedvd** [reply]
It's works !!! Well done and thx you ^^ Honestly, it's not intuitive at the first sight but now you tell me how its work, that make sense.
Thx you a lot ^^
